Database/Repositories/Implementations/TournamentsRepository.cs

```diff
@@ -105,12 +111,53 @@ public async Task<ICollection<Tournament>> GetAsync(
 
         if (ruleset.HasValue)
         {
-            query = query.Where(x => x.Ruleset == ruleset.Value);
+            query = query.Where(t => t.Ruleset == ruleset.Value);
+        }
+
+        if (!string.IsNullOrEmpty(name))
+        {
+            query = query.WhereNameOrAbbreviation(name);
+        }
+
+        if (dateMin.HasValue)
+        {
+            query = query.Where(t => t.StartTime > dateMin.Value);
```

Comment:
```suggestion
            query = query.Where(t => t.StartTime >= dateMin.Value);
```

Reply:
FTR, **I agree**, however the osu! mp match timestamps have a precision down to the second so if this ever actually effected anyone's search I would be shocked beyond belief

Reply:
I'm sure there's one instance out of 100k matches lol